test/test-network/systemd-networkd-tests.py

```diff
@@ -3084,6 +3087,12 @@ def test_vxlan(self):
         print(output)
         self.assertIn('inet6 2002:da8:1:0:1034:56ff:fe78:9abc/64 scope global dynamic', output)
 
+        output = check_output('ip -d link show vxlan-external')
+        print(output)
+        self.assertIn('vxlan id 0', output)
```

Comment:
This will fail with iproute2 older than 6.4.0, specifically https://github.com/iproute2/iproute2/commit/1215e9d3862387353d8672296cb4c6c16e8cbb72.

Please make the check string `' id 0 '`